### PR TITLE
docs(config) stats.orphanModules doc improved slightly

### PR DESCRIPTION
**Pull request by EugeneHlushko**
A small follow up to #3362 https://github.com/webpack/webpack.js.org/pull/3362/files#r357783625

### CODE REVIEW

File: src/content/configuration/stats.md [hunk 1]
```diff
@@ -320,7 +320,7 @@ module.exports = {
 
 `boolean = false`
 
-Tells `stats` whether to hide `orphan` modules. A module is an `orphan` if it is not included in any chunk. Orphans modules are now hidden by default in `Stats`.
+Tells `stats` whether to hide `orphan` modules. A module is an `orphan` if it is not included in any chunk. Orphan modules are hidden by default in `stats`.
```
Comment (montogeek) [inline src/content/configuration/stats.md]:
```suggestion
Tells `stats` whether to hide `orphan` modules. A module is `orphan` if it is not included in any chunk. Orphan modules are hidden by default in `stats`.
```

Comment (jeffin143) [reply src/content/configuration/stats.md]: Should we mention , from webpack 5

Now could be replaced by webpack 5 ig

Comment (EugeneHlushko) [reply src/content/configuration/stats.md]: I think `an` is good, just ran it through my native speaker mate.

@jeffin143 It says webpack 5 docs in the left hand side, so we should stop doing these mentions probably

**Comment by jeffin143** [comment]
Just for reference #2437

**Pull request merged**